src/Sylius/Bundle/FixturesBundle/DataFixtures/ORM/LoadTaxonsData.php

```diff
@@ -27,21 +27,21 @@ class LoadTaxonsData extends DataFixture
     public function load(ObjectManager $manager)
     {
         $manager->persist($this->createTaxons(
-            'CATEGORY', [$this->defaultLocale => 'Category', 'es_ES' => 'Categoria'],
+            'category', [$this->defaultLocale => 'Category', 'es_ES' => 'Categoria'],
```

Comment:
If you're changing this to lowercase you have fix [this line](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml#L262) too ;)